File: Samples/C#.Net/CSharpSDKTest/ISmpHelpDlg.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using SalesMatePlusLib; 

namespace CSharpSDKTest
{
    public partial class ISmpHelpDlg : Form
    {
        public ISmpHelpDlg()
        {
            InitializeComponent();
        }

        private void TipOfTheDayDlgbtn_Click(object sender, EventArgs e)
        {
            ISmpHelp m_SmpHelp;

            m_SmpHelp = new SmpHelpClass();

            m_SmpHelp.InvokeTipOfTheDayDlg();
        }


        private void InvokeRegisterbtn_Click(object sender, EventArgs e)
        {
            ISmpHelp m_SmpHelp;

            m_SmpHelp = new SmpHelpClass();

            m_SmpHelp.InvokeRegisterDlg();
        }

        private void AboutBoxbtn_Click(object sender, EventArgs e)
        {
            ISmpHelp m_SmpHelp;

            m_SmpHelp = new SmpHelpClass();

            m_SmpHelp.InvokeAboutBoxDlg();
        }

        private void HelTopicsbtn_Click(object sender, EventArgs e)
        {
            ISmpHelp m_SmpHelp;

            m_SmpHelp = new SmpHelpClass();

            string strHlpFliPath;

            strHlpFliPath = m_SmpHelp.HelpFilePath;

            m_SmpHelp.InvokeHelpTopics(strHlpFliPath);
        }

        private void MenuShortcutKeybtn_Click(object sender, EventArgs e)
        {
            ISmpHelp m_SmpHelp;

            m_SmpHelp = new SmpHelpClass();

            m_SmpHelp.InvokeMenuShortcutKeyDlg();
        }
    }
}

File: Samples/VB.Net/VBDotNet/ISmpHelpDlg.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports SalesMatePlusLib

Namespace CSharpSDKTest
	Partial Public Class ISmpHelpDlg
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub TipOfTheDayDlgbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles TipOfTheDayDlgbtn.Click
			Dim m_SmpHelp As ISmpHelp

			m_SmpHelp = New SmpHelpClass()

			m_SmpHelp.InvokeTipOfTheDayDlg()
		End Sub


		Private Sub InvokeRegisterbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles InvokeRegisterbtn.Click
			Dim m_SmpHelp As ISmpHelp

			m_SmpHelp = New SmpHelpClass()

			m_SmpHelp.InvokeRegisterDlg()
		End Sub

		Private Sub AboutBoxbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles AboutBoxbtn.Click
			Dim m_SmpHelp As ISmpHelp

			m_SmpHelp = New SmpHelpClass()

			m_SmpHelp.InvokeAboutBoxDlg()
		End Sub

		Private Sub HelTopicsbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles HelTopicsbtn.Click
			Dim m_SmpHelp As ISmpHelp

			m_SmpHelp = New SmpHelpClass()

			Dim strHlpFliPath As String

			strHlpFliPath = m_SmpHelp.HelpFilePath

			m_SmpHelp.InvokeHelpTopics(strHlpFliPath)
		End Sub

		Private Sub MenuShortcutKeybtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles MenuShortcutKeybtn.Click
			Dim m_SmpHelp As ISmpHelp

			m_SmpHelp = New SmpHelpClass()

			m_SmpHelp.InvokeMenuShortcutKeyDlg()
		End Sub
	End Class
End Namespace